pkg/util/metadata/metadata.go

```diff
@@ -77,6 +77,15 @@ type Metadata struct {
 	// .. and other fields we don't care about.  Expand as necessary.
 }
 
+type IMetadata interface {
+	GetInstanceID(order string) (string, error)
```

Comment:
GetInstanceID, GetAvailabilityZone this is not necessary, we can modify csi code to get metadata as ccm is doing, to have consistency.